sql/core/src/test/java/test/org/apache/spark/sql/JavaSparkSessionSuite.java

```diff
@@ -54,4 +55,19 @@ public void config() {
       Assert.assertEquals(spark.conf().get(e.getKey()), e.getValue().toString());
     }
   }
+
+  @Test
+  public void sqlParameters() {
```

Comment:
do we need a java test? Adding a overload that takes a java map is very trivial...

Reply:
I removed it.